package-lock.json

```diff
@@ -778,11 +777,6 @@
       "resolved": "https://registry.npmjs.org/@edsilv/http-status-codes/-/http-status-codes-1.0.3.tgz",
       "integrity": "sha512-HLK2FS5sZqxPqD53D6hhZxC6C8THTVwlyZDZ7J0iWsrB8JmMA69m/CQuNKZc1kki9WSVeck2fXna26NL0SE7cg=="
     },
-    "node_modules/@edsilv/jquery-plugins": {
```

Comment:
I notice that removing the gallery component seems to have also removed the dependency on the jquery plugins. We should double-check that there is not code in the gallery component that depends on these, or else we may need to explicitly add this back as a direct dependency of UV.

Reply:
I've added it as a direct dependency for now.  I'm still investigating if it is essential to operations.  Will hopefully have some more clear information on that soon.

Reply:
I believe that it is currently essential -- at very least, when I search for the first of the plugins (the `checkboxButton` function), I see that the gallery component uses it when multi-select is enabled. The problem is that I don't think anyone knows how to enable multi-selection functionality in order to properly test this.

Regardless, this is not the only plugin that is used, nor are the plugins used exclusively in GalleryComponent -- for example, the `horizontalMargins` plugin is used in quite a few places, including OpenSeadragonCenterPanel and HeaderPanel.

Looks to me like getting rid of this dependency would be quite a bit of work. For now, leaving it as an explicit dependency makes sense to me so we don't forget about it.